backend/corpora/dataset_processing/download.py

```diff
@@ -1,11 +1,21 @@
 import logging
 import threading
 import requests
-
-from ..common.corpora_orm import DbDatasetProcessingStatus, UploadStatus
-from ..common.entities import Dataset
-from ..common.utils.db_utils import db_session_manager
-from ..common.utils.math_utils import MB
+import os
+import sys
+
+try:
+    from ..common.corpora_orm import DbDatasetProcessingStatus, UploadStatus
+    from ..common.entities import Dataset
+    from ..common.utils.db_utils import db_session_manager
+    from ..common.utils.math_utils import MB
+except ValueError:
```

Comment:
thats the error that gets thrown for relative imports
```
ValueError: attempted relative import beyond top-level package
```
It also throws ModuleNotFoundError, I can catch that instead if you think it would be more clear

Reply:
ModuleNotFoundError would be more clear

Reply:
Looks like gets caught on the wrong level/doesnt work. I can add a comment explaining why we are catching the ValueError